src/gnatss/main.py

```diff
@@ -184,24 +219,33 @@ def get_transmit_times(
 def get_reply_times(
     cleaned_travel_times: pd.DataFrame,
     all_gps_solutions: pd.DataFrame,
+    rph_data: pd.DataFrame,
     gps_sigma_limit: float,
     transponder_ids: List[str],
+    atd_offsets: NDArray[Shape["3"], Float],
+    array_center: ArrayCenter,
```

Comment:
So these two inputs are actually very specific to the computation for rph data, so I think it would make more sense to make these `Optional` rather than a required field.

Reply:
Done!